ramls/items-in-transit.json

```diff
@@ -77,6 +92,74 @@
                 "description": "The location code"
               }
             }
+          },
+          "request": {
+            "description": "The first open item's request in the queue",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "requestType": {
+                "type": "string",
+                "description": "The type of the request"
+              },
+              "requestDate": {
+                "description": "Date the request was made",
+                "type": "string",
+                "format": "date-time"
+              },
+              "requestExpirationDate": {
+                "description": "Date when the request expires",
+                "type": "string",
+                "format": "date-time"
+              },
+              "requestPickupServicePointName": {
+                "description": "The name of the request pickup service point",
+                "type": "string"
+              },
+              "requestPatronGroup": {
+                "type": "string",
+                "description": "The patron group of the request"
+              }
+            }
+          },
+          "loan": {
+            "description": "The effective location of the item",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "loan": {
```

Comment:
it duplicates the name of the parent object, so this property will have `loan.loan` path, is it expected?

Reply:
It is typo. Chaged